rest_server/src/controllers/item_controller.js

```diff
@@ -1,17 +1,25 @@
 // Copyright (c) Microsoft Corporation.
 // Licensed under the MIT License.
 const { isNil } = require('lodash');
-const { MarketplaceItem } = require('../models');
+const { MarketplaceItem, ItemCategory } = require('../models');
 const asyncHandler = require('./async_handler');
 const yaml = require('js-yaml');
 const protocolValidator = require('../utils/protocol');
 const error = require('../models/error');
 
-function checkReadPermission(userInfo, item) {
+const OFFICIAL_EXAMPLE = 'official example';
```

Comment:
>official example [](http://example.com/codeflow?start=26&length=16)

see the same value in both this file and marketplace_item.js, so can we define it at single place?

Reply:
Moved them to file `item_category.js`